spec/factories/funding_methods.rb

```diff
@@ -6,6 +6,14 @@
     amount { Faker::Number.number(digits: 5) }
     funding_type { FUNDING_TYPE_ENUMS[:bursary] }
 
+    trait :bursary do
```

Comment:
https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#enum-traits

Not really needed

Reply:
i always forget this - thanks!